signals.go

```diff
@@ -8,20 +8,20 @@ import (
 	"github.com/go-task/task/v3/internal/logger"
 )
 
-const interruptSignalsCount = 3
+const maxInterruptSignals = 3
 
 // NOTE(@andreynering): This function intercepts SIGINT and SIGTERM signals
 // so the Task process is not killed immediately and processes running have
 // time to do cleanup work.
 func (e *Executor) InterceptInterruptSignals() {
-	ch := make(chan os.Signal, interruptSignalsCount)
+	ch := make(chan os.Signal, maxInterruptSignals)
 	signal.Notify(ch, os.Interrupt, syscall.SIGTERM)
 
 	go func() {
-		for i := range interruptSignalsCount {
+		for i := 0; i < maxInterruptSignals; i++ {
```

Comment:
These loops are [functionally the same](https://goplay.tools/snippet/nyFZquJ-MXh) and I personally prefer the newer `for i := range maxInterruptSignals` syntax.

No objection to the renaming.

Reply:
I agree